src/views/service_panel.dot

```diff
@@ -12,11 +12,7 @@
   {{? this.categories.length > 0 }}
   <div class="service_panel__categories">
     {{~ this.categories:category:index }}
-      {{?
-        !this.mustDeployCategories ||
-        (this.mustDeployCategories && !this.isDeployed && index < 12) ||
-        (this.mustDeployCategories && this.isDeployed)
-      }}
+      {{? index < 12 }}
```

Comment:
I guess you could even remove this test for now, but it's no big deal :)

Reply:
consider it a "garde-fous" in case someone touches categories.yml